app/helpers/routing_helper.rb

```diff
@@ -2,37 +2,40 @@
 require 'json'
 
 module RoutingHelper
-  def self.seconds_to_minsec(sec)
+  def self.format_seconds_as_minsec(sec)
     format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
   end
 
   def self.valid_coordinates(coordinates)
```

Comment:
Something that I didn't notice at first: This predicate should probably end with a `?`:
```suggestion
  def self.valid_coordinates?(coordinates)
```
I am 100% sure, but I think this is according to ruby style recommendations